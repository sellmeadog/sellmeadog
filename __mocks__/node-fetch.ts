export * from 'node-fetch';
export default jest.fn();
