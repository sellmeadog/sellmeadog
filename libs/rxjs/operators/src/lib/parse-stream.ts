import { Response } from 'node-fetch';
import {
  asyncScheduler,
  filter,
  map,
  mergeMap,
  Observable,
  OperatorFunction,
  scheduled,
} from 'rxjs';
import split = require('split2');

export function parse_stream<T = unknown>(
  matchExp: RegExp,
  mapFn: (data: string, matchExp: RegExp) => T = parse_stream_map_fn
) {
  return (source$: Observable<Response>) =>
    source$.pipe(
      lines(),
      filter((line) => matchExp.test(line)),
      map((line) => mapFn(line, matchExp))
    );
}

function lines(): OperatorFunction<Response, string> {
  return mergeMap((response) =>
    scheduled(response.body.pipe(split()), asyncScheduler)
  );
}

function parse_stream_map_fn<T = unknown>(data: string, matchExp: RegExp): T {
  return JSON.parse(data.match(matchExp)[1]);
}
