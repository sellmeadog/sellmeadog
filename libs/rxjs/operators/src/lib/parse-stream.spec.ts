import { Response } from 'node-fetch';
import { parse_stream } from './parse-stream';
import { createReadStream } from 'fs';
import { resolve } from 'path';
import { lastValueFrom, of, toArray } from 'rxjs';

describe('parse_stream', () => {
  it('should parse file line by line', async () => {
    const rs = createReadStream(resolve(__dirname, 'oracle-cards.json'));
    const response = new Response(rs);

    const data = await lastValueFrom(
      of(response).pipe(parse_stream(/({.+}),?/), toArray())
    );

    expect(data).toMatchSnapshot();
  });
});
