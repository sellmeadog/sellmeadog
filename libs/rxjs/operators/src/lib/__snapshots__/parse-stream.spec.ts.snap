// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse_stream should parse file line by line 1`] = `
Array [
  Object {
    "artist": "Terese Nielsen",
    "artist_ids": Array [
      "eb55171c-2342-45f4-a503-2d5a75baf752",
    ],
    "booster": true,
    "border_color": "white",
    "card_back_id": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "cardmarket_id": 3081,
    "cmc": 3,
    "collector_number": "319",
    "color_identity": Array [],
    "colors": Array [],
    "digital": false,
    "edhrec_rank": 2557,
    "finishes": Array [
      "nonfoil",
      "foil",
    ],
    "flavor_text": "The warriors fought against the paralyzing waves until even their thoughts froze in place.",
    "foil": true,
    "frame": "1997",
    "full_art": false,
    "games": Array [
      "paper",
      "mtgo",
    ],
    "highres_image": true,
    "id": "86bf43b1-8d4e-4759-bb2d-0b2e03ba7012",
    "illustration_id": "6f8b3b2c-252f-4f95-b621-712c82be38b5",
    "image_status": "highres_scan",
    "image_uris": Object {
      "art_crop": "https://c1.scryfall.com/file/scryfall-cards/art_crop/front/8/6/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012.jpg?1562242171",
      "border_crop": "https://c1.scryfall.com/file/scryfall-cards/border_crop/front/8/6/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012.jpg?1562242171",
      "large": "https://c1.scryfall.com/file/scryfall-cards/large/front/8/6/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012.jpg?1562242171",
      "normal": "https://c1.scryfall.com/file/scryfall-cards/normal/front/8/6/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012.jpg?1562242171",
      "png": "https://c1.scryfall.com/file/scryfall-cards/png/front/8/6/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012.png?1562242171",
      "small": "https://c1.scryfall.com/file/scryfall-cards/small/front/8/6/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012.jpg?1562242171",
    },
    "keywords": Array [],
    "lang": "en",
    "layout": "normal",
    "legalities": Object {
      "alchemy": "not_legal",
      "brawl": "not_legal",
      "commander": "legal",
      "duel": "legal",
      "future": "not_legal",
      "gladiator": "not_legal",
      "historic": "not_legal",
      "historicbrawl": "not_legal",
      "legacy": "legal",
      "modern": "not_legal",
      "oldschool": "not_legal",
      "pauper": "not_legal",
      "paupercommander": "not_legal",
      "penny": "not_legal",
      "pioneer": "not_legal",
      "premodern": "legal",
      "standard": "not_legal",
      "vintage": "legal",
    },
    "mana_cost": "{3}",
    "mtgo_foil_id": 15871,
    "mtgo_id": 15870,
    "multiverse_ids": Array [
      15862,
    ],
    "name": "Static Orb",
    "nonfoil": true,
    "object": "card",
    "oracle_id": "0004ebd0-dfd6-4276-b4a6-de0003e94237",
    "oracle_text": "As long as Static Orb is untapped, players can't untap more than two permanents during their untap steps.",
    "oversized": false,
    "prices": Object {
      "eur": "9.00",
      "eur_foil": "0.39",
      "tix": "0.49",
      "usd": "21.80",
      "usd_etched": null,
      "usd_foil": "325.00",
    },
    "prints_search_uri": "https://api.scryfall.com/cards/search?order=released&q=oracleid%3A0004ebd0-dfd6-4276-b4a6-de0003e94237&unique=prints",
    "promo": false,
    "rarity": "rare",
    "related_uris": Object {
      "edhrec": "https://edhrec.com/route/?cc=Static+Orb",
      "gatherer": "https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=15862",
      "mtgtop8": "https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Static+Orb",
      "tcgplayer_infinite_articles": "https://infinite.tcgplayer.com/search?contentMode=article&game=magic&partner=scryfall&q=Static+Orb&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
      "tcgplayer_infinite_decks": "https://infinite.tcgplayer.com/search?contentMode=deck&game=magic&partner=scryfall&q=Static+Orb&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
    },
    "released_at": "2001-04-11",
    "reprint": true,
    "reserved": false,
    "rulings_uri": "https://api.scryfall.com/cards/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012/rulings",
    "scryfall_set_uri": "https://scryfall.com/sets/7ed?utm_source=api",
    "scryfall_uri": "https://scryfall.com/card/7ed/319/static-orb?utm_source=api",
    "set": "7ed",
    "set_id": "230f38aa-9511-4db8-a3aa-aeddbc3f7bb9",
    "set_name": "Seventh Edition",
    "set_search_uri": "https://api.scryfall.com/cards/search?order=set&q=e%3A7ed&unique=prints",
    "set_type": "core",
    "set_uri": "https://api.scryfall.com/sets/230f38aa-9511-4db8-a3aa-aeddbc3f7bb9",
    "story_spotlight": false,
    "tcgplayer_id": 3094,
    "textless": false,
    "type_line": "Artifact",
    "uri": "https://api.scryfall.com/cards/86bf43b1-8d4e-4759-bb2d-0b2e03ba7012",
    "variation": false,
  },
  Object {
    "artist": "Steven Belledin",
    "artist_ids": Array [
      "f07d73b9-52a0-4fe5-858b-61f7b42174a5",
    ],
    "booster": true,
    "border_color": "black",
    "card_back_id": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "cardmarket_id": 262945,
    "cmc": 1,
    "collector_number": "71",
    "color_identity": Array [
      "U",
    ],
    "colors": Array [
      "U",
    ],
    "digital": false,
    "edhrec_rank": 21486,
    "finishes": Array [
      "nonfoil",
      "foil",
    ],
    "flavor_text": "They call it \\"stitcher's anesthesia,\\" a spell to deaden the senses while the mad doctors begin their grisly work.",
    "foil": true,
    "frame": "2003",
    "full_art": false,
    "games": Array [
      "paper",
      "mtgo",
    ],
    "highres_image": true,
    "id": "7050735c-b232-47a6-a342-01795bfd0d46",
    "illustration_id": "3c9e9355-4dcd-411c-b85f-47386e94854b",
    "image_status": "highres_scan",
    "image_uris": Object {
      "art_crop": "https://c1.scryfall.com/file/scryfall-cards/art_crop/front/7/0/7050735c-b232-47a6-a342-01795bfd0d46.jpg?1562830795",
      "border_crop": "https://c1.scryfall.com/file/scryfall-cards/border_crop/front/7/0/7050735c-b232-47a6-a342-01795bfd0d46.jpg?1562830795",
      "large": "https://c1.scryfall.com/file/scryfall-cards/large/front/7/0/7050735c-b232-47a6-a342-01795bfd0d46.jpg?1562830795",
      "normal": "https://c1.scryfall.com/file/scryfall-cards/normal/front/7/0/7050735c-b232-47a6-a342-01795bfd0d46.jpg?1562830795",
      "png": "https://c1.scryfall.com/file/scryfall-cards/png/front/7/0/7050735c-b232-47a6-a342-01795bfd0d46.png?1562830795",
      "small": "https://c1.scryfall.com/file/scryfall-cards/small/front/7/0/7050735c-b232-47a6-a342-01795bfd0d46.jpg?1562830795",
    },
    "keywords": Array [
      "Enchant",
    ],
    "lang": "en",
    "layout": "normal",
    "legalities": Object {
      "alchemy": "not_legal",
      "brawl": "not_legal",
      "commander": "legal",
      "duel": "legal",
      "future": "not_legal",
      "gladiator": "not_legal",
      "historic": "not_legal",
      "historicbrawl": "not_legal",
      "legacy": "legal",
      "modern": "legal",
      "oldschool": "not_legal",
      "pauper": "legal",
      "paupercommander": "legal",
      "penny": "legal",
      "pioneer": "legal",
      "premodern": "not_legal",
      "standard": "not_legal",
      "vintage": "legal",
    },
    "mana_cost": "{U}",
    "mtgo_foil_id": 49284,
    "mtgo_id": 49283,
    "multiverse_ids": Array [
      370780,
    ],
    "name": "Sensory Deprivation",
    "nonfoil": true,
    "object": "card",
    "oracle_id": "0006faf6-7a61-426c-9034-579f2cfcfa83",
    "oracle_text": "Enchant creature
Enchanted creature gets -3/-0.",
    "oversized": false,
    "prices": Object {
      "eur": "0.02",
      "eur_foil": "0.05",
      "tix": "0.04",
      "usd": "0.02",
      "usd_etched": null,
      "usd_foil": "0.09",
    },
    "prints_search_uri": "https://api.scryfall.com/cards/search?order=released&q=oracleid%3A0006faf6-7a61-426c-9034-579f2cfcfa83&unique=prints",
    "promo": false,
    "rarity": "common",
    "related_uris": Object {
      "edhrec": "https://edhrec.com/route/?cc=Sensory+Deprivation",
      "gatherer": "https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=370780",
      "mtgtop8": "https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Sensory+Deprivation",
      "tcgplayer_infinite_articles": "https://infinite.tcgplayer.com/search?contentMode=article&game=magic&partner=scryfall&q=Sensory+Deprivation&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
      "tcgplayer_infinite_decks": "https://infinite.tcgplayer.com/search?contentMode=deck&game=magic&partner=scryfall&q=Sensory+Deprivation&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
    },
    "released_at": "2013-07-19",
    "reprint": true,
    "reserved": false,
    "rulings_uri": "https://api.scryfall.com/cards/7050735c-b232-47a6-a342-01795bfd0d46/rulings",
    "scryfall_set_uri": "https://scryfall.com/sets/m14?utm_source=api",
    "scryfall_uri": "https://scryfall.com/card/m14/71/sensory-deprivation?utm_source=api",
    "set": "m14",
    "set_id": "e03ee1c0-ecd2-4fcc-ac3c-e8fdb103a847",
    "set_name": "Magic 2014",
    "set_search_uri": "https://api.scryfall.com/cards/search?order=set&q=e%3Am14&unique=prints",
    "set_type": "core",
    "set_uri": "https://api.scryfall.com/sets/e03ee1c0-ecd2-4fcc-ac3c-e8fdb103a847",
    "story_spotlight": false,
    "tcgplayer_id": 69965,
    "textless": false,
    "type_line": "Enchantment â€” Aura",
    "uri": "https://api.scryfall.com/cards/7050735c-b232-47a6-a342-01795bfd0d46",
    "variation": false,
  },
  Object {
    "artist": "Jonas De Ro",
    "artist_ids": Array [
      "561ebf9e-8d93-4b57-8156-8826d0c19601",
    ],
    "booster": false,
    "border_color": "black",
    "card_back_id": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "cardmarket_id": 391692,
    "cmc": 2,
    "collector_number": "34",
    "color_identity": Array [
      "G",
    ],
    "colors": Array [
      "G",
    ],
    "digital": false,
    "edhrec_rank": 3992,
    "finishes": Array [
      "nonfoil",
    ],
    "foil": false,
    "frame": "2015",
    "full_art": false,
    "games": Array [
      "paper",
      "mtgo",
    ],
    "highres_image": true,
    "id": "e718b21b-46d1-4844-985c-52745657b1ac",
    "illustration_id": "279f7cef-a81c-4b43-98a0-937f3c355b5e",
    "image_status": "highres_scan",
    "image_uris": Object {
      "art_crop": "https://c1.scryfall.com/file/scryfall-cards/art_crop/front/e/7/e718b21b-46d1-4844-985c-52745657b1ac.jpg?1568003608",
      "border_crop": "https://c1.scryfall.com/file/scryfall-cards/border_crop/front/e/7/e718b21b-46d1-4844-985c-52745657b1ac.jpg?1568003608",
      "large": "https://c1.scryfall.com/file/scryfall-cards/large/front/e/7/e718b21b-46d1-4844-985c-52745657b1ac.jpg?1568003608",
      "normal": "https://c1.scryfall.com/file/scryfall-cards/normal/front/e/7/e718b21b-46d1-4844-985c-52745657b1ac.jpg?1568003608",
      "png": "https://c1.scryfall.com/file/scryfall-cards/png/front/e/7/e718b21b-46d1-4844-985c-52745657b1ac.png?1568003608",
      "small": "https://c1.scryfall.com/file/scryfall-cards/small/front/e/7/e718b21b-46d1-4844-985c-52745657b1ac.jpg?1568003608",
    },
    "keywords": Array [
      "Entwine",
    ],
    "lang": "en",
    "layout": "normal",
    "legalities": Object {
      "alchemy": "not_legal",
      "brawl": "not_legal",
      "commander": "legal",
      "duel": "legal",
      "future": "not_legal",
      "gladiator": "not_legal",
      "historic": "not_legal",
      "historicbrawl": "not_legal",
      "legacy": "legal",
      "modern": "not_legal",
      "oldschool": "not_legal",
      "pauper": "not_legal",
      "paupercommander": "not_legal",
      "penny": "not_legal",
      "pioneer": "not_legal",
      "premodern": "not_legal",
      "standard": "not_legal",
      "vintage": "legal",
    },
    "mana_cost": "{G}{G}",
    "mtgo_id": 77122,
    "multiverse_ids": Array [
      470580,
    ],
    "name": "Road of Return",
    "nonfoil": true,
    "object": "card",
    "oracle_id": "0007c283-5b7a-4c00-9ca1-b455c8dff8c3",
    "oracle_text": "Choose one â€”
â€¢ Return target permanent card from your graveyard to your hand.
â€¢ Put your commander into your hand from the command zone.
Entwine {2} (Choose both if you pay the entwine cost.)",
    "oversized": false,
    "preview": Object {
      "previewed_at": "2019-08-05",
      "source": "Magicshibby",
      "source_uri": "https://www.youtube.com/watch?v=39MC2wBcf00&t=7m51s",
    },
    "prices": Object {
      "eur": "0.58",
      "eur_foil": null,
      "tix": "0.71",
      "usd": "0.29",
      "usd_etched": null,
      "usd_foil": null,
    },
    "prints_search_uri": "https://api.scryfall.com/cards/search?order=released&q=oracleid%3A0007c283-5b7a-4c00-9ca1-b455c8dff8c3&unique=prints",
    "promo": false,
    "rarity": "rare",
    "related_uris": Object {
      "edhrec": "https://edhrec.com/route/?cc=Road+of+Return",
      "gatherer": "https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=470580",
      "mtgtop8": "https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Road+of+Return",
      "tcgplayer_infinite_articles": "https://infinite.tcgplayer.com/search?contentMode=article&game=magic&partner=scryfall&q=Road+of+Return&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
      "tcgplayer_infinite_decks": "https://infinite.tcgplayer.com/search?contentMode=deck&game=magic&partner=scryfall&q=Road+of+Return&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
    },
    "released_at": "2019-08-23",
    "reprint": false,
    "reserved": false,
    "rulings_uri": "https://api.scryfall.com/cards/e718b21b-46d1-4844-985c-52745657b1ac/rulings",
    "scryfall_set_uri": "https://scryfall.com/sets/c19?utm_source=api",
    "scryfall_uri": "https://scryfall.com/card/c19/34/road-of-return?utm_source=api",
    "security_stamp": "oval",
    "set": "c19",
    "set_id": "0fa3ccbb-d86d-4a2e-a55e-c4979c4feeb2",
    "set_name": "Commander 2019",
    "set_search_uri": "https://api.scryfall.com/cards/search?order=set&q=e%3Ac19&unique=prints",
    "set_type": "commander",
    "set_uri": "https://api.scryfall.com/sets/0fa3ccbb-d86d-4a2e-a55e-c4979c4feeb2",
    "story_spotlight": false,
    "tcgplayer_id": 196536,
    "textless": false,
    "type_line": "Sorcery",
    "uri": "https://api.scryfall.com/cards/e718b21b-46d1-4844-985c-52745657b1ac",
    "variation": false,
  },
  Object {
    "artist": "John Matson",
    "artist_ids": Array [
      "a1685587-4b55-446b-b420-c37b202ed3f1",
    ],
    "booster": true,
    "border_color": "white",
    "card_back_id": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "cardmarket_id": 12551,
    "cmc": 2,
    "collector_number": "100",
    "color_identity": Array [
      "U",
    ],
    "colors": Array [
      "U",
    ],
    "digital": false,
    "edhrec_rank": 12284,
    "finishes": Array [
      "nonfoil",
      "foil",
    ],
    "flavor_text": "Storm crow descending, winter unending. Storm crow departing, summer is starting.",
    "foil": true,
    "frame": "2003",
    "full_art": false,
    "games": Array [
      "paper",
      "mtgo",
    ],
    "highres_image": true,
    "id": "036ef8c9-72ac-46ce-af07-83b79d736538",
    "illustration_id": "d01aa92b-0582-4e1e-a7b0-737b2ad4e462",
    "image_status": "highres_scan",
    "image_uris": Object {
      "art_crop": "https://c1.scryfall.com/file/scryfall-cards/art_crop/front/0/3/036ef8c9-72ac-46ce-af07-83b79d736538.jpg?1562730661",
      "border_crop": "https://c1.scryfall.com/file/scryfall-cards/border_crop/front/0/3/036ef8c9-72ac-46ce-af07-83b79d736538.jpg?1562730661",
      "large": "https://c1.scryfall.com/file/scryfall-cards/large/front/0/3/036ef8c9-72ac-46ce-af07-83b79d736538.jpg?1562730661",
      "normal": "https://c1.scryfall.com/file/scryfall-cards/normal/front/0/3/036ef8c9-72ac-46ce-af07-83b79d736538.jpg?1562730661",
      "png": "https://c1.scryfall.com/file/scryfall-cards/png/front/0/3/036ef8c9-72ac-46ce-af07-83b79d736538.png?1562730661",
      "small": "https://c1.scryfall.com/file/scryfall-cards/small/front/0/3/036ef8c9-72ac-46ce-af07-83b79d736538.jpg?1562730661",
    },
    "keywords": Array [
      "Flying",
    ],
    "lang": "en",
    "layout": "normal",
    "legalities": Object {
      "alchemy": "not_legal",
      "brawl": "not_legal",
      "commander": "legal",
      "duel": "legal",
      "future": "not_legal",
      "gladiator": "not_legal",
      "historic": "not_legal",
      "historicbrawl": "not_legal",
      "legacy": "legal",
      "modern": "legal",
      "oldschool": "not_legal",
      "pauper": "legal",
      "paupercommander": "legal",
      "penny": "legal",
      "pioneer": "not_legal",
      "premodern": "legal",
      "standard": "not_legal",
      "vintage": "legal",
    },
    "mana_cost": "{1}{U}",
    "mtgo_foil_id": 22610,
    "mtgo_id": 22609,
    "multiverse_ids": Array [
      83282,
    ],
    "name": "Storm Crow",
    "nonfoil": true,
    "object": "card",
    "oracle_id": "000d5588-5a4c-434e-988d-396632ade42c",
    "oracle_text": "Flying (This creature can't be blocked except by creatures with flying or reach.)",
    "oversized": false,
    "power": "1",
    "prices": Object {
      "eur": "0.02",
      "eur_foil": "1.89",
      "tix": "0.03",
      "usd": "0.18",
      "usd_etched": null,
      "usd_foil": "2.49",
    },
    "prints_search_uri": "https://api.scryfall.com/cards/search?order=released&q=oracleid%3A000d5588-5a4c-434e-988d-396632ade42c&unique=prints",
    "promo": false,
    "rarity": "common",
    "related_uris": Object {
      "edhrec": "https://edhrec.com/route/?cc=Storm+Crow",
      "gatherer": "https://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=83282",
      "mtgtop8": "https://mtgtop8.com/search?MD_check=1&SB_check=1&cards=Storm+Crow",
      "tcgplayer_infinite_articles": "https://infinite.tcgplayer.com/search?contentMode=article&game=magic&partner=scryfall&q=Storm+Crow&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
      "tcgplayer_infinite_decks": "https://infinite.tcgplayer.com/search?contentMode=deck&game=magic&partner=scryfall&q=Storm+Crow&utm_campaign=affiliate&utm_medium=api&utm_source=scryfall",
    },
    "released_at": "2005-07-29",
    "reprint": true,
    "reserved": false,
    "rulings_uri": "https://api.scryfall.com/cards/036ef8c9-72ac-46ce-af07-83b79d736538/rulings",
    "scryfall_set_uri": "https://scryfall.com/sets/9ed?utm_source=api",
    "scryfall_uri": "https://scryfall.com/card/9ed/100/storm-crow?utm_source=api",
    "set": "9ed",
    "set_id": "e70c8572-4732-4e92-a140-b4e3c1c84c93",
    "set_name": "Ninth Edition",
    "set_search_uri": "https://api.scryfall.com/cards/search?order=set&q=e%3A9ed&unique=prints",
    "set_type": "core",
    "set_uri": "https://api.scryfall.com/sets/e70c8572-4732-4e92-a140-b4e3c1c84c93",
    "story_spotlight": false,
    "tcgplayer_id": 12835,
    "textless": false,
    "toughness": "2",
    "type_line": "Creature â€” Bird",
    "uri": "https://api.scryfall.com/cards/036ef8c9-72ac-46ce-af07-83b79d736538",
    "variation": false,
  },
]
`;
